src/Nuget.PackageIndex.VisualStudio/Analyzers/ProjectTargetFrameworkProviderExporter.cs

```diff
@@ -175,6 +176,16 @@ private void OnAfterSolutionClosing()
             ClearCache();
         }
 
+        private void OnProjectRemoved(Project project)
+        {
+            if (project == null)
```

Comment:
I doubt this event ever passes null, and if it does its probably a bug we should handle because something weird is going one.